packages/material-ui/src/TableFooter/TableFooter.js

```diff
@@ -1,22 +1,33 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import clsx from 'clsx';
-import withStyles from '../styles/withStyles';
+import makeStyles from '../styles/makeStyles';
+import useThemeProps from '../styles/useThemeProps';
 import Tablelvl2Context from '../Table/Tablelvl2Context';
+import muiComponent from '../utils/muiComponent';
 
-export const styles = {
+const tablelvl2 = {
+  variant: 'footer',
+};
+
+const styles = {
```

Comment:
Why did we need to change this?

Reply:
We don't need to change it but we don't need to export it. What's the use case for exporting it? I can revert.

Reply:
It was used before for our docsgen. What's the use case for removing it when that changes requires changes somewhere else? It's just PR noise.

Reply:
Yes, it's why I have reverted the change yesterday.